index.html

```diff
@@ -34,7 +34,7 @@
 </p>
 
 {% highlight bash %}
-zypper install yast2-devtools rubygem-yast-rake git-core
+zypper install yast2-devtools ruby2.1-rubygem-yast-rake git-core
```

Comment:
I thought about adding this paragraph below.

> Note that in the code above, we use a variable "ruby_version" to store the 
version of ruby installed on the system. This is because the version of 
yast-rake package that you need depends of the ruby version.
Finally be careful with the space before and after =, there should be no none.